app/controllers/api/v1/search_results_controller.rb

```diff
@@ -22,7 +22,7 @@ def create
       private
 
       def upload_form_params
-        params.permit(:search_engine, :csv_file)
+        params.permit(:search_engine, :csv_file, :client_id, :client_secret)
```

Comment:
`:client_id` and `:client_secret` are only sent for the sign up endpoint. After that the access token is used to authorize every request 🔑

Reply:
@olivierobert I will also add it to the sign-in endpoint, as the response for sign-up will not include the token yet (because we must verify the user's email first) 🙏